prow/cmd/pipeline/dev.yaml

```diff
@@ -0,0 +1,119 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: prow-pipeline
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: prow-pipeline
+  template:
+    metadata:
+      labels:
+        app: prow-pipeline
+    spec:
+      serviceAccount: prow-pipeline
+      containers:
+      - name: pipeline
+        image: gcr.io/k8s-testimages/pipeline:latest  # Note: not gcr.io/k8s-prow for dev
+        imagePullPolicy: Always  # Good practice for dev/debugging, bad for prod
+        args:
+        - --tls-cert-file=/etc/tls/tls.crt
+        - --tls-private-key-file=/etc/tls/tls.key
+        ports:
+        - containerPort: 8443
+          name: validator-http
+          protocol: TCP
+        volumeMounts:
+        - name: tls
+          mountPath: /etc/tls
+      volumes:
+      - name: tls
+        secret:
+          secretName: prow-pipeline
```

Comment:
I think we can get rid of the ports, volume, and tls args now that the admission controller is omitted.

Reply:
done